index.qmd

```diff
@@ -84,48 +84,102 @@ accessibility tests, and more, consult our [Guides](#guides)
 
 ### Installation
 
-To install the workbench, make sure you have a working version of R and 
-pandoc/RStudio installed (see [the workbench setup instructions for 
+:::: {.panel-tabset}
+
+#### Within R
+
+To install the workbench, make sure you have a working version of R and
+pandoc/RStudio installed (see [the workbench setup instructions for
 details](https://carpentries.github.io/sandpaper-docs/)).
 
-::: {.callout-tip collapse='true'}
+From there, you can install the workbench packages and their dependencies from
```

Comment:
Workbench here should be capitalised.

Reply:
@froggleston can do &mdash; will push a fix now. Above it there is the section

https://github.com/carpentries/workbench/blob/fd2090bf1505c2268a4f54d55bdf69fcf4bd9804/index.qmd#L87-L89

from https://github.com/carpentries/workbench/commit/ae93d4f983722f1278e9f88997426579dba7f48e. Should all instances of text be 'The Workbench'?

Reply:
@froggleston Ah okay. That differs from the rest of the text that exists in the `index.qmd` at the moment where 'The Workbench` is used throughout.

e.g.

* https://github.com/carpentries/workbench/blob/fd2090bf1505c2268a4f54d55bdf69fcf4bd9804/index.qmd#L58
* https://github.com/carpentries/workbench/blob/fd2090bf1505c2268a4f54d55bdf69fcf4bd9804/index.qmd#L68
* https://github.com/carpentries/workbench/blob/fd2090bf1505c2268a4f54d55bdf69fcf4bd9804/index.qmd#L81
* https://github.com/carpentries/workbench/blob/fd2090bf1505c2268a4f54d55bdf69fcf4bd9804/index.qmd#L98
* https://github.com/carpentries/workbench/blob/fd2090bf1505c2268a4f54d55bdf69fcf4bd9804/index.qmd#L133-L134

So I assume you want those all changed to 'the Workbench' too?

Reply:
> Ah okay. That differs from the rest of the text that exists in the index.qmd at the moment where 'The Workbench' is used throughout.

I can understand the confusion: our style guidance on this has been evolving (or perhaps better to admit it's been non-existent..?) 😅 I prefer either "The Carpentries Workbench" (uppercase "T" in "The") or "the Workbench" (lowercase "t" in "the") for short.